### PR TITLE
Explore Feed: add icons to peek controller context menu

### PR DESCRIPTION
**Pull request by evieluvsrainbows**
**Phabricator:** N/A

### Notes
* Adds icons in the context menu for the peek experience in the Explore Feed
* This uses SF Symbols, the built-in icons library. I tried to use the most appropriate icons for each action.

### Test Steps
1. Run the app, hold down on an article in Explore Feed
2. Look at the new icons! :smiley:

### Screenshots/Videos

Simulator Screenshot - iPhone 15 Pro Max - 2024-08-28 at 23 06 20
Simulator Screenshot - iPhone 15 Pro Max - 2024-08-28 at 23 07 46

### CODE REVIEW

**Comment by tonisevener** [comment]
Hi @evelynharthbrooke, thank you for the improvement! I have created an associated Phabricator task so that our designer can take a look - https://phabricator.wikimedia.org/T374282. I am putting these changes through design review before we merge and will update this PR if any tweaks are needed.

**Review by tonisevener — CHANGES_REQUESTED** [review]
Thanks for the PR @evelynharthbrooke! This has been approved by design. I have one minor request from code review. Let us know if you have any questions!

**Comment by evelynharthbrooke** [comment]
> Thanks for the PR @evelynharthbrooke! This has been approved by design. I have one minor request from code review. Let us know if you have any questions!

I will get that done as soon as possible once i have a bit of free time! :)

**Comment by evelynharthbrooke** [comment]
I will update this PR with the recommended changes as soon as the PR that adds support for Xcode 16 is merged, as my Mac is on Sequoia and Xcode 16 now, and I don't want to push a whole bunch of irrelevant changes to my branches.

File: Wikipedia/Code/ArticleViewController+LinkPreviewing.swift [hunk 1]
```diff
@@ -30,7 +30,7 @@ extension ArticleViewController: ArticleContextMenuPresenting, WKUIDelegate {
     var contextMenuItems: [UIAction] {
         // Read action
         let readActionTitle = WMFLocalizedString("button-read-now", value: "Read now", comment: "Read now button text used in various places.")
-        let readAction = UIAction(title: readActionTitle, handler: { (action) in
+        let readAction = UIAction(title: readActionTitle, image: UIImage(systemName: "book"), handler: { (action) in
```
[the remark this answers is not in the document]
Comment (evelynharthbrooke) [reply Wikipedia/Code/ArticleViewController+LinkPreviewing.swift]: Done!

**Comment by evelynharthbrooke** [comment]
@tonisevener I have made the requested changes. :)

**Review by tonisevener — APPROVED** [review]

**Comment by tonisevener** [comment]
Looks great, thank you @evelynharthbrooke!

**Pull request merged**